### ISSUE TITLE
Support for alternative locales

### ISSUE THREAD

**Issue by davidcurrie** [WASdev/ci.docker#102]
The base Ubuntu image only includes the locales C, C.UTF-8 and POSIX. We should document how to create images with alternative locales. The proposal is to include commented out steps in our production-install scripts and also document how to layer on top of our Docker Hub images to add locales. For example:

```
FROM websphere-liberty:webProfile7
RUN apt-get update \
  && apt-get install -y language-pack-pt-base \
  && rm -rf /var/lib/apt/lists/*
ENV LANG pt_BR.UTF-8
```

**Comment by davidcurrie** [comment]
Partly resolved by #104. Other half pending https://github.com/docker-library/docs/pull/780.

**Comment by davidcurrie** [comment]
Docker Hub docs updated.
